src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj

```diff
@@ -6,7 +6,7 @@
     <DefineConstants>$(DefineConstants);BUILDING_PKCS</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
-    <NoWarn>$(NoWarn);CA5384</NoWarn>
+    <NoWarn>$(NoWarn);CA5384;SYSLIB0057</NoWarn>
```

Comment:
You both fixed things and suppressed it?

Assuming this is a multitargeting problem, I'd solve it with something like

```C#
internal static X509Certificate2 LoadCertificate(...)
{
#if NET_9_OR_GREATER
    return X509CertificateLoader.LoadCertificate(...);
#else
    if (X509Certificate.GetCertContentType(...) != X509ContentType.Cert)
    {
        // Might warrant a real message
        throw new CryptographicException();
    }

    return new X509Certificate2(...);
#endif
}
```

_technically_ there's a ToC/ToU for that from a file, so if it needs a file it could read it in... or just accept the mild ToC/ToU.

Reply:
The suppress was left over from me getting back to a buildable state. It can just be removed.